### ISSUE TITLE
Can't build walls or floors

### ISSUE THREAD

**Issue by Gouty** [tgstation/tgstation#34027]
Wow, I've seen players in game complain about this for the last 24 hours on almost every round, even had people contact me on discord about it and not one of them thought to make a report.

Currently walls cannot be created, adding metal to a wall girder causes it to disappear, and using an RCD does nothing (but does use units, make the noise and give you the text "You build a wall.")

Just found out an RCD in "deconstruct" mode DOES create a wall

If you put down a lattice and attempt to tile it then the floor and the lattice disappears.  If you then put another rod down on the same spot both a lattice and floor plate appear.

RCDs cannot make floors

Basically it is all badly broken.

**Comment by DaxDupont** [comment]
https://gfycat.com/SharpLeafyLeech

**Comment by Harakoniwarhawk** [comment]
Metal foam grenades seem to be broken too, repeated uses and had no affect at all, linked in some way?

**Comment by ninjanomnom** [comment]
I completely forgot about the changeturf part of placeontop() while fixing it's behavior earlier, I'll have a fix up momentarily
